pkg/asset/manifests/tectonic.go

```diff
@@ -16,12 +20,17 @@ import (
 
 const (
 	tectonicManifestDir = "tectonic"
+	// TODO: Verify this is expected os creds file
```

Comment:
[Looks like][1] we can drop this comment?

[1]: https://github.com/openshift/installer/pull/427/files#r227364281